framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -117,6 +118,49 @@
     drillProperties = ImmutableMap.copyOf(properties);
   }
 
+
+ /**
+ * Change to something like "Check if the value passed to the -s parameter (test definition sources) can be 
+ * expanded as a regex expression and use that to construct the list of test definitions
+ *
+ * @return list of test definition sources to execute
+ */
+  public static String[] getTestDefSources() {
+
+    String[] testDefSources = null;
+    try {
+     testDefSources = TestDriver.cmdParam.sources.split(",");
+    } catch (Exception e) {
+      testDefSources = new String[] { "" }; //Look at the default location for test definition files
+    }
+    for (String testDefSource : testDefSources) {
+      testDefSource = getAbsolutePath(testDefSource, TestDriver.drillTestDataDir);
+      File testDefSourceFile = new File(testDefSource);
+      List<File> testQueryFolders = new ArrayList<>();
+      if (!testDefSourceFile.exists()) {
+        //try regex then exit if failure
+        LOG.info("Directory " + CWD + "/" + TestDriver.drillTestDataDir+ " value !");
+        File drillTestDataDir = new File(CWD + "/" + TestDriver.drillTestDataDir);
+        if (drillTestDataDir.exists()) {
+              testQueryFolders.addAll(searchFolders(drillTestDataDir,
+                            testDefSource));
+        }
+      if(testQueryFolders.size()==0){
```

Comment:
Change to if(testDefinitionList.isEmpty()). Also add a log indicating that no matching test definitions were found.